  INFO root [2021-09-29 17:39:40,345] - 
  INFO root [2021-09-29 17:39:40,346] - -----------------------------------------------------------------------------------------


  INFO root [2021-09-29 17:39:40,347] - Resmgr::initialize: KBE_ROOT=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/
  INFO root [2021-09-29 17:39:40,347] - Resmgr::initialize: KBE_RES_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/res/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/scripts/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/res/
  INFO root [2021-09-29 17:39:40,347] - Resmgr::initialize: KBE_BIN_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/bin/server/
  INFO root [2021-09-29 17:39:40,385] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.31.145 ( 192.168.31.145:63504 )
  INFO root [2021-09-29 17:39:40,385] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.31.145:63504, SOMAXCONN=511.
  INFO root [2021-09-29 17:39:40,385] - server-configs:
  INFO root [2021-09-29 17:39:40,385] - 	internalTcpAddr : 192.168.31.145:63504
  INFO root [2021-09-29 17:39:40,385] - 	componentID : 6000
  WARN root [2021-09-29 17:39:40,385] - invalid UID(0) <= 0, please check UID for environment! automatically set to 12169.
  INFO root [2021-09-29 17:39:40,484] - Components::findLogger: found logger, addr:192.168.31.145:63501
  INFO root [2021-09-29 17:39:40,484] - Components::addComponent: reset orderLog, uid=12169!
  INFO root [2021-09-29 17:39:40,485] - Components::addComponent[logger], uid=12169, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2021-09-29 17:39:40,489] - ---- cellappmgr Version: 2.5.10. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 5BD150375E8109EF4863A845CB8F8699. Config: Debug X64. Built: 14:52:54 Sep 29 2021. AppID: 6000. UID: 12169. PID: 19132 ----
  INFO root [2021-09-29 17:39:40,489] - Server Administrator: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz with 16710936kB RAM
  INFO root [2021-09-29 17:39:40,490] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2021-09-29 17:39:40,490] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2021-09-29 17:39:40,769] - ---- cellappmgr is running ----
 DEBUG root [2021-09-29 17:39:40,769] - Components::process(): Request for the process of identity...
  WARN root [2021-09-29 17:39:46,974] - Forwarding logs to logger[192.168.31.145:63501/0/0/0]...
 ERROR root [2021-09-29 18:52:22,015] - Components::removeComponentByChannel: baseappmgr : 5000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=5100008800226, lastReceivedTime=5099891408586, inactivityExceptionPeriod=599500192)
 ERROR root [2021-09-29 18:53:22,094] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=timedout)! Channel(timestamp=5100609588930, lastReceivedTime=5099890181446, inactivityExceptionPeriod=599500192)
  INFO root [2021-09-29 18:53:24,178] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2021-09-29 19:20:23,502] - 
  INFO root [2021-09-29 19:20:23,518] - -----------------------------------------------------------------------------------------


  INFO root [2021-09-29 19:20:23,519] - Resmgr::initialize: KBE_ROOT=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/
  INFO root [2021-09-29 19:20:23,519] - Resmgr::initialize: KBE_RES_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/res/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/scripts/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/res/
  INFO root [2021-09-29 19:20:23,519] - Resmgr::initialize: KBE_BIN_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/bin/server/
  INFO root [2021-09-29 19:20:23,558] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.31.145 ( 192.168.31.145:50943 )
  INFO root [2021-09-29 19:20:23,558] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.31.145:50943, SOMAXCONN=511.
  INFO root [2021-09-29 19:20:23,558] - server-configs:
  INFO root [2021-09-29 19:20:23,558] - 	internalTcpAddr : 192.168.31.145:50943
  INFO root [2021-09-29 19:20:23,558] - 	componentID : 6000
  WARN root [2021-09-29 19:20:23,558] - invalid UID(0) <= 0, please check UID for environment! automatically set to 12169.
  INFO root [2021-09-29 19:20:23,596] - Components::findLogger: found logger, addr:192.168.31.145:50938
  INFO root [2021-09-29 19:20:23,596] - Components::addComponent: reset orderLog, uid=12169!
  INFO root [2021-09-29 19:20:23,596] - Components::addComponent[logger], uid=12169, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2021-09-29 19:20:23,600] - ---- cellappmgr Version: 2.5.10. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 5BD150375E8109EF4863A845CB8F8699. Config: Debug X64. Built: 14:52:54 Sep 29 2021. AppID: 6000. UID: 12169. PID: 3880 ----
  INFO root [2021-09-29 19:20:23,601] - Server Administrator: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz with 16710936kB RAM
  INFO root [2021-09-29 19:20:23,601] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2021-09-29 19:20:23,601] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2021-09-29 19:20:23,886] - ---- cellappmgr is running ----
 DEBUG root [2021-09-29 19:20:23,886] - Components::process(): Request for the process of identity...
  WARN root [2021-09-29 19:20:29,993] - Forwarding logs to logger[192.168.31.145:50938/0/0/0]...
 ERROR root [2021-09-29 19:21:39,287] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=13691757597, lastReceivedTime=13506813597, inactivityExceptionPeriod=599500000)
   OFF root [2021-09-29 19:21:39,288] - The following logs sent to logger failed:
  WARN root [2021-09-29 19:21:39,288] - ==> [2021-09-29 19:21:39 287] TCPPacketReceiver::processPendingEvents(192.168.31.145:50938): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2021-09-29 19:21:39,289] - ==> [2021-09-29 19:21:39 287] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=13691757597, lastReceivedTime=13506813597, inactivityExceptionPeriod=599500000)
  INFO root [2021-09-29 19:21:45,504] - 
  INFO root [2021-09-29 19:21:45,504] - -----------------------------------------------------------------------------------------


  INFO root [2021-09-29 19:21:45,505] - Resmgr::initialize: KBE_ROOT=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/
  INFO root [2021-09-29 19:21:45,505] - Resmgr::initialize: KBE_RES_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/res/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/scripts/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/res/
  INFO root [2021-09-29 19:21:45,506] - Resmgr::initialize: KBE_BIN_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/bin/server/
  INFO root [2021-09-29 19:21:45,542] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.31.145 ( 192.168.31.145:51018 )
  INFO root [2021-09-29 19:21:45,542] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.31.145:51018, SOMAXCONN=511.
  INFO root [2021-09-29 19:21:45,542] - server-configs:
  INFO root [2021-09-29 19:21:45,542] - 	internalTcpAddr : 192.168.31.145:51018
  INFO root [2021-09-29 19:21:45,542] - 	componentID : 6000
  WARN root [2021-09-29 19:21:45,542] - invalid UID(0) <= 0, please check UID for environment! automatically set to 12169.
  INFO root [2021-09-29 19:21:45,602] - Components::findLogger: found logger, addr:192.168.31.145:51014
  INFO root [2021-09-29 19:21:45,602] - Components::addComponent: reset orderLog, uid=12169!
  INFO root [2021-09-29 19:21:45,602] - Components::addComponent[logger], uid=12169, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2021-09-29 19:21:45,606] - ---- cellappmgr Version: 2.5.10. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 5BD150375E8109EF4863A845CB8F8699. Config: Debug X64. Built: 14:52:54 Sep 29 2021. AppID: 6000. UID: 12169. PID: 4176 ----
  INFO root [2021-09-29 19:21:45,606] - Server Administrator: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz with 16710936kB RAM
  INFO root [2021-09-29 19:21:45,607] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2021-09-29 19:21:45,607] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2021-09-29 19:21:45,880] - ---- cellappmgr is running ----
 DEBUG root [2021-09-29 19:21:45,880] - Components::process(): Request for the process of identity...
  WARN root [2021-09-29 19:21:51,996] - Forwarding logs to logger[192.168.31.145:51014/0/0/0]...
 ERROR root [2021-09-29 19:22:06,847] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=13967361035, lastReceivedTime=13754922007, inactivityExceptionPeriod=599499745)
   OFF root [2021-09-29 19:22:06,848] - The following logs sent to logger failed:
  WARN root [2021-09-29 19:22:06,848] - ==> [2021-09-29 19:22:06 847] TCPPacketReceiver::processPendingEvents(192.168.31.145:51014): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2021-09-29 19:22:06,848] - ==> [2021-09-29 19:22:06 848] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=13967361035, lastReceivedTime=13754922007, inactivityExceptionPeriod=599499745)
